powerline/listers/i3wm.py

```diff
@@ -0,0 +1,78 @@
+# vim:fileencoding=utf-8:noet
+from __future__ import (unicode_literals, division, absolute_import, print_function)
+
+import re
+
+from powerline.theme import requires_segment_info
+from powerline.lib.shell import run_cmd
+
+
+conn = None
+
+
+@requires_segment_info
+def output_lister(pl, segment_info):
+	'''List all outputs in segment_info format'''
+
+	outputs = re.findall(
+		'^([0-9A-Za-z-]+) connected',
```

Comment:
>I would also highly suggest this to abstract away into a function used by both this file and script.

what script? I thought `re.findall` was "smart enough" to do that internally but I will do that.